lib/child_process.js

```diff
@@ -440,6 +452,8 @@ function normalizeSpawnArguments(file, args, options) {
   }
 
   // Validate the shell, if present.
+  // const shell = options.hasOwnProperty('shell') ? options.shell : undefined;
```

Comment:
It's not clear if this is commented out code or part of the comment above

Reply:
cleared them out, thanks!